R/tag.R

```diff
@@ -241,3 +241,21 @@ tag_markdown <- function(x) {
   x$val <- markdown_if_active(x$val, x)
   tag_value(x)
 }
+
+#' @export
+#' @rdname roxy_tag
+tag_markdown_with_sections <- function(x) {
+  x$val <- markdown_if_active(x$val, x, sections = TRUE)
+  if (length(x$val) == 1 && x$val[[1]] == "") {
+    roxy_tag_warning(x, "requires a value")
+  }
+  for (i in seq_along(x$val)) {
+    if (!rdComplete(x$val[i])) {
```

Comment:
Why doesn't this match `tag_markdown()`?

Reply:
Because `x$val` is a vector now, First element is the regular content, the rest are the sections. But again, we can make this more explicit by returning a list.

Reply:
But `tag_markdown()` doesn't perform this check at all. Why not?

Reply:
It does I think, it calls `tag_value()`, that does these checks.

Reply:
Ah got it.